### ISSUE TITLE
Tmux on Alacritty gets sent alt+f4 keys after closing another application

### ISSUE THREAD

**Issue by butla** [alacritty/alacritty#3243]
I've only noticed that recently, but I'm not sure if it started with 0.4.1 or a bit earlier.

When alacritty's windows is the next window after the current one (would be jumped to if I pressed alt+tab once), and I close the current window, whatever program is running in alacritty gets sent some keystrokes. If I have Vim insert mode enabled, it adds `<C-S-F4>` (is this ctrl + shift + f4?)

Only happens if Alacritty is running Tmux. Doesn't happen with Tmux on a different shell (xfce4-terminal). I've checked env vars differences between alacritty and xfce4 terminal running tmux, and the only vars that are different are PATH, TMUX, TMUX_PANE and WINDOWID.

### System

OS: Linux, Manjaro / Kubuntu 18.04
Version: 0.4.1
Linux/BSD: X11, KDE and XFCE

### Logs

Keyboard and bindings:
```
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1102, tv_nsec: 104745055 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 28, state: Released, virtual_keycode: Some(Return), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1104, tv_nsec: 683655239 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(ALT) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1106, tv_nsec: 69311777 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Pressed, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1106, tv_nsec: 246495928 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1106, tv_nsec: 356327355 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged((empty)) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1107, tv_nsec: 354523374 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(true) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: LogicalPosition { x: 1355.0, y: 229.0 }, modifiers: (empty) } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1107, tv_nsec: 357112862 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(false) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(true) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: LogicalPosition { x: 1355.0, y: 229.0 }, modifiers: (empty) } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorEntered { device_id: DeviceId(X(DeviceId(2))) } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: LogicalPosition { x: 1355.0, y: 229.0 }, modifiers: (empty) } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1107, tv_nsec: 359774266 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1107, tv_nsec: 361200763 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1107, tv_nsec: 366183699 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(ALT) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 612542066 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: false } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 612636214 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Pressed, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 764406109 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(false) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 765404439 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorLeft { device_id: DeviceId(X(DeviceId(2))) } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 765506311 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 779680489 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1109, tv_nsec: 868140959 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Pressed, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1110, tv_nsec: 641342847 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1110, tv_nsec: 728136471 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged((empty)) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1111, tv_nsec: 447099786 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(false) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1111, tv_nsec: 449807716 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1111, tv_nsec: 463232741 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(ALT) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1113, tv_nsec: 243205170 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Pressed, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1113, tv_nsec: 396646286 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1113, tv_nsec: 503262135 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 42, state: Pressed, virtual_keycode: Some(LShift), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(SHIFT | ALT) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1114, tv_nsec: 725881094 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Pressed, virtual_keycode: Some(Tab), modifiers: SHIFT | ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1115, tv_nsec: 361422078 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: SHIFT | ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1115, tv_nsec: 465031351 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 42, state: Released, virtual_keycode: Some(LShift), modifiers: SHIFT | ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(ALT) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1115, tv_nsec: 754870114 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged((empty)) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1115, tv_nsec: 799990753 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(ALT) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 387088010 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 62, state: Pressed, virtual_keycode: Some(F4), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 869157545 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(true) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: LogicalPosition { x: 1355.0, y: 229.0 }, modifiers: ALT } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 62, state: Pressed, virtual_keycode: Some(F4), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorEntered { device_id: DeviceId(X(DeviceId(2))) } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: LogicalPosition { x: 1355.0, y: 229.0 }, modifiers: ALT } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 898737646 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: UserEvent(MouseCursorDirty)
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 899018736 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: UserEvent(Wakeup)
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 910324923 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 911788499 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 929122460 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 62, state: Released, virtual_keycode: Some(F4), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(false) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(true) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorMoved { device_id: DeviceId(X(DeviceId(2))), position: LogicalPosition { x: 1355.0, y: 229.0 }, modifiers: ALT } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 961830301 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1117, tv_nsec: 978593847 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged((empty)) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: (empty) }, is_synthetic: false } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1118, tv_nsec: 85764132 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(ALT) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Pressed, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: false } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 53790479 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Pressed, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: KeyboardInput { device_id: DeviceId(X(DeviceId(3))), input: KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }, is_synthetic: true } }
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(false) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 210818689 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: CursorLeft { device_id: DeviceId(X(DeviceId(2))) } }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 211025038 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 227983848 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 15, state: Released, virtual_keycode: Some(Tab), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 278281962 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 56, state: Released, virtual_keycode: Some(LAlt), modifiers: ALT }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged((empty)) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 805574944 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: WindowEvent { window_id: WindowId(X(WindowId(12582914))), event: Focused(false) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 808048631 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1121, tv_nsec: 811471817 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 29, state: Pressed, virtual_keycode: Some(LControl), modifiers: (empty) }) }
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: ModifiersChanged(CTRL) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
[2020-01-22 19:37] [INFO] glutin event: NewEvents(WaitCancelled { start: Instant { tv_sec: 1122, tv_nsec: 312172620 }, requested_resume: None })
[2020-01-22 19:37] [INFO] glutin event: DeviceEvent { device_id: DeviceId(X(DeviceId(16))), event: Key(KeyboardInput { scancode: 46, state: Pressed, virtual_keycode: Some(C), modifiers: CTRL }) }
[2020-01-22 19:37] [INFO] glutin event: MainEventsCleared
[2020-01-22 19:37] [INFO] glutin event: RedrawEventsCleared
```

**Comment by kchibisov** [comment]
Could you try the latest master?

**Comment by butla** [comment]
Ok, I'll try building from source

**Comment by butla** [comment]
Doesn't seem to happen on the latest version. Ok, closing this and waiting for the next release :)

**Comment by kchibisov** [comment]
It was likely fixed by c1664b348d97d6ab1a1a6956e1823ca70e64351c
